pupa/scrape/event.py

```diff
@@ -62,7 +62,7 @@ def __init__(self, name, start_time, timezone, location, *,
         self.description = description
         self.status = status
         self.classification = classification
-        self.location = {"name": location, "note": "", "coordinates": None}
+        self.location = location
```

Comment:
```diff
@@ -62,7 +62,7 @@ class Event(BaseModel, SourceMixin, AssociatedLinkMixin, LinkMixin):
self.description = description
self.status = status
self.classification = classification
-        self.location = {"name": location, "note": "", "coordinates": None}
+        self.location = location
```

Why this change? Seems like that forces folks to create this dict? I'd either
set it to None and handle that elsewhere or keep the full dict here. Pehaps
changing the name of that kwarg to location_name would be better?